### PR TITLE
[DEV APPROVED]Ruby upgrade

### PR DESCRIPTION
**Pull request by jriga**
[Card 7745](https://moneyadviceservice.tpondemand.com/entity/7745)

**Upgrade**
 - [x] Rails to 4.1.16
 - [x] Ruby to 2.3.3

**Dependencies**
[Comfy Upgrade](https://github.com/moneyadviceservice/comfortable-mexican-sofa/pull/22)
[Script with ruby 2.3.3](https://github.com/moneyadviceservice/scripts/pull/841)

### CODE REVIEW

File: app/models/comfy/cms/category.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@ class Comfy::Cms::Category < ActiveRecord::Base
   validates_presence_of :label, :title_en, :title_cy
   validates_uniqueness_of :label, :title_en, :title_cy, scope: :site_id
 
-  belongs_to :small_image, class: Comfy::Cms::File
-  belongs_to :large_image, class: Comfy::Cms::File
+  belongs_to :small_image, class_name: Comfy::Cms::File
```
Comment (jongilbraith) [inline app/models/comfy/cms/category.rb]: I think typically the class name is passed as a string when used as this option.

File: Gemfile [hunk 2]
```diff
@@ -2,8 +2,8 @@ source 'http://gems.dev.mas.local'
 source 'https://rubygems.org'
 
 gem 'bugsnag'
-gem 'rails', '4.1.1'
-gem 'sqlite3'
+gem 'rails', '4.1.16'
+#gem 'sqlite3'
```
Comment (jongilbraith) [inline Gemfile]: It this isn't needed, just remove the line entirely.

**Comment by jongilbraith** [comment]
👍 👍

File: Gemfile (hunk 2 above)
Comment (jriga) [inline Gemfile]: ♻️ not needed and removed

File: app/models/comfy/cms/category.rb (hunk 1 above)
Comment (jriga) [inline app/models/comfy/cms/category.rb]: ♻️

**Comment by tomas-stefano** [comment]
👍

**Pull request merged**